pkg/cmd/pulumi/stack_history.go

```diff
@@ -72,8 +73,10 @@ This command displays data about previous updates for a stack.`,
 		"Show secret values when listing config instead of displaying blinded values")
 	cmd.PersistentFlags().BoolVarP(
 		&jsonOut, "json", "j", false, "Emit output as JSON")
-	cmd.PersistentFlags().IntVarP(
-		&limit, "limit", "l", 0, "Limit the number of entries returned, defaults to all")
```

Comment:
Do we care about breaking anyone currently using `--limit`? Should we keep it as a hidden flag for now?

Reply:
@justinvp that change was merged just a few days ago and hasn't been released 😄 

Taking the opportunity to alter the functionality now before it becomes a breaking change.